great_expectations/checkpoint/util.py

```diff
@@ -206,6 +206,8 @@ def get_substituted_validation_dict(
             validation_dict.get("runtime_configuration", {}),
             dedup=True,
         ),
+        "include_rendered_content": validation_dict.get("include_rendered_content")
+        or substituted_runtime_config.get("include_rendered_content"),
```

Comment:
Same note here - if possible, can we keep the type a `bool` and not create the possibility of a null value?

Reply:
I have also update this to use `False` if both `validation_dict` and `subsituted_runtime_config` return `None` for `include_rendered_content`. ✅